modules/installation-deployment-manager-vpc.adoc

```diff
@@ -67,9 +54,8 @@ def GenerateConfig(context):
                 }]
             }, {
                 'name': context.properties['infra_id'] + '-nat-worker',
-                'natIpAllocateOption': 'MANUAL_ONLY',
-                'natIps': ['$(ref.' + context.properties['infra_id'] + '-worker-nat-ip.selfLink)'],
-                'minPortsPerVm': 128,
+                'natIpAllocateOption': 'AUTO_ONLY',
+                'minPortsPerVm': 512,
                 'sourceSubnetworkIpRangesToNat': 'LIST_OF_SUBNETWORKS',
                 'subnetworks': [{
                     'name': '$(ref.' + context.properties['infra_id'] + '-worker-subnet.selfLink)',
```

Comment:
If we cover the VPC network creation, that would help build a better user experience. So could you please add a section to introduce the way to create VPC network in host project? The procedure is like this:
```javascript
$ cat <<EOF >01_vpc.yaml
imports:
- path: 01_vpc.py

resources:
- name: cluster-vpc
  type: 01_vpc.py
  properties:
    infra_id: '${INFRA_ID}' 
    region: '${REGION}' 
    master_subnet_cidr: '${MASTER_SUBNET_CIDR}' 
    worker_subnet_cidr: '${WORKER_SUBNET_CIDR}' 
EOF

$ gcloud deployment-manager deployments create ${INFRA_ID}-vpc --config 01_vpc.yaml --project ${HOST_PROJECT}     --account ${HOST_PROJECT_ACCOUNT}
```